### PR TITLE
Link to kerberos add-on was leading to a 404 page

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by stefano-ottolenghi** [comment]
Uhm, I'm not sure what's happening here. I can confirm the live link is broken in the 4.4 version of the docs. At the same time, the link is correct in the adoc file, and the variable it uses is correctly set both in `preview.yml` and in `publish.yml` (the link is outdated, but there is a redirect, so whatever). And in fact, when building locally `4.4`, the link does render correctly:
Screenshot from 2022-11-09 12-10-59

It does only seem to happen in the live docs. Might there be something happening with Antora/building here? @recrwplay

File: common-content/modules/ROOT/partials/client-applications.adoc [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ include::{python-examples}/test_kerberos_auth_example.py[tags=kerberos-auth]
 
 [NOTE]
 --
-The Kerberos authentication token can only be understood by the server if the server has the link:{kerberos-add-on-base-uri}[Kerberos Add-on] installed.
+The Kerberos authentication token can only be understood by the server if the server has the link:/docs/kerberos-add-on/current/[Kerberos Add-on] installed.
```
Comment (recrwplay) [inline common-content/modules/ROOT/partials/client-applications.adoc:1073]:
```suggestion
The Kerberos authentication token can only be understood by the server if the server has the link:{neo4j-docs-base-uri}/kerberos-add-on[Kerberos Add-on] installed.
```

The `neo4j-docs-base-uri` has just been added (this morning!) and will always resolve correctly whether you are building locally with _preview.yml_ or publishing with _publish.yml_.

**Comment by recrwplay** [comment]
> It does only seem to happen in the live docs. Might there be something happening with Antora/building here? @recrwplay

Correct, the published docs are built from the _publish.yml_ playbook in the 5.0 branch. The attribute in this link is not in that playbook, and not defined in any other file used by that build, so it doesn't resolve.

**Comment by lidiazuin** [comment]
@recrwplay are you going to change that in the playbook for 4.x? Should I close this PR then?

**Comment by recrwplay** [comment]
@lidiazuin yes, I'll make sure the playbooks in all the branches are up to date so this can be closed.

**Comment by stefano-ottolenghi** [comment]
But I think Neil's suggestion to Lidia's original change still needs to be applied, right? To use `{neo4j-docs-base-uri}`.

**Comment by lidiazuin** [comment]
True!! I missed that. Thanks for flagging it @stefano-ottolenghi

**Pull request closed without merging**